docs/developer-docs/multi-chain/chain-key-tokens/ckbtc/making-transactions.mdx

```diff
@@ -75,7 +75,7 @@ There are several different ways to interact with ckBTC:
 </div>
 
 :::info
-Since this is a real bitcoin transaction, the finality time will take about 2 hours and will be charged the Bitcoin network's transaction fees.
+Since this is a real bitcoin transaction, the finality time will take about 10 minutes and will be charged the Bitcoin network's transaction fees.
```

Comment:
This is not correct! It now takes 6 (and no longer 12) confirmations, so it should say 1 hour here.